thread_options.h

```diff
@@ -428,6 +429,7 @@ struct thread_options_pack {
 
 	uint32_t perc_rand[DDIR_RWDIR_CNT];
 
+	uint16_t start_offset_percent;
```

Comment:
Just make this uint32_t, please. Ideally we don't want anything unaligned in the packed structure, that means that 32-bit entries should be 32-bit aligne. If you add a 16-bit value, then that shifts everything 16-bits.

Fix that up, and the typo Sitsofe noticed, and I think we're ready to get this merged!